spec/features/favorites/pet_favorites_indicator_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe 'Favorite Pets Indicator', type: :feature do
+  describe "as a visitor visiting any page in the application" do
+    it "I can see the number of favorite pets on the navigation bar" do
+      visit '/shelters'
+
+      expect(page).to have_content("Favorite Pets: 0")
+
+      visit '/pets'
+
+      expect(page).to have_content("Favorite Pets: 0")
+    end
```

Comment:
As we discussed, I think we can eventually move this entire test into a separate spec file for testing the nav bar and verifying that it appears on every page and acts as expected. But if we don't end up having time, I think this is fine also.